src/components/PersonTable/PersonTable.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+
+import { Person } from '../../types';
+import { PersonInfo } from '../PersonInfo/PersonInfo';
+
+type Props = {
+  people: Person[];
+};
+
+export const PersonTable: React.FC<Props> = ({ people }) => {
+  const { slug = '' } = useParams();
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          {['Name', 'Sex', 'Born', 'Died', 'Mother', 'Father'].map(column => (
+            <th key={column}>{column}</th>
```

Comment:
create a constant for the the table headers, it will improve the readability and maintainability of your code